packages/openapi-ts/src/utils/write/services.ts

```diff
@@ -200,9 +200,21 @@ const toOperationComment = (operation: Operation): Comments => {
   return comment;
 };
 
-const toRequestOptions = (operation: Operation, onClientImport?: OnImport) => {
+const toRequestOptions = (
+  client: Client,
+  operation: Operation,
+  onClientImport: OnImport | undefined,
+  responseType: string,
+) => {
   const config = getConfig();
 
+  const hasResponseTransformer =
+    operation.results.length > 0 &&
+    operation.results[0].imports.length > 0 &&
+    operation.results[0].properties.length === 0 &&
+    client.types[responseType]?.['hasTransformer'];
```

Comment:
TODO: I might have overcooked these bools, did this early on and can't remember why those  middle 2 results need to be there

TODO: Also some discussion on what to do if there are multiple success results in the list, as it implies either a all-of union type, or several success status codes which we could be discriminating on